src/pages/Bridge/IssueForm/index.tsx

```diff
@@ -5,19 +5,20 @@ import {
   useSelector
 } from 'react-redux';
 import { useForm } from 'react-hook-form';
-import { useTranslation } from 'react-i18next';
-import Big from 'big.js';
-import clsx from 'clsx';
+import { useQuery } from 'react-query';
 import {
   useErrorHandler,
   withErrorBoundary
 } from 'react-error-boundary';
-import { AccountId } from '@polkadot/types/interfaces';
+import { useTranslation } from 'react-i18next';
+import Big from 'big.js';
+import clsx from 'clsx';
 import {
   Issue,
   newMonetaryAmount,
   GovernanceUnit
 } from '@interlay/interbtc-api';
+import { IssueLimits } from '@interlay/interbtc-api/build/src/parachain/issue';
```

Comment:
Can you confirm this returns exactly the same data as the previous call to `window.bridge.vaults.getVaultsWithIssuableTokens()`? I'm a bit nervous about switching out the way in which we call and validate the vaults data this close to release?

Reply:
Yes, I confirmed it. Dom suggested using it instead of `window.bridge.vaults.getVaultsWithIssuableTokens`. As a result, the workflow got simpler.